K8S.md

```diff
@@ -36,7 +36,12 @@ SSH into a node
 ```                                                                        
 cluster-up/ssh.sh node01                                                   
 ```                                                                        
-                                                                           
+
+Use slim provider (without pre-pulled images of the optional components such as CDI, CNAO etc)
+```
+export KUBEVIRT_SLIM=true
```

Comment:
I am not sure if this is the best place for this information - reading the doc as a whole this update seems to break the flow and is not really relevant?

Reply:
I can move it to cluster-up/cluster/K8S.md
and / or add a "misc" section in that file, just before the new text
as also the console help below it, is pretty misc

wdyt ?

Reply:
> I think the console information is useful to have there for someone starting out with kubevirtci who may be having issues connecting to the nodes over ssh.

You mean to move `Attach to node console with screen and pty` to their as well ?
If so it can be done on a follow PR please, as it is not related to this PR
I will move the related part to that file.

Reply:
Sorry I meant to keep the console information where it is as it is useful there.